Account/Status.ts

```diff
@@ -0,0 +1,22 @@
+import { isly } from "isly"
+
+export type Status = {
+	mode: "active" | "frozen" | "closed"
+	reason?: "overdraft" | "other" | string
```

Comment:
Suggestion: `reason?: "overdraft" | "other" | (string & Record<never, never>)` to fool typescript and get type suggestions, now Typescript will simplify this type to just `string`

Reply:
Also, put this type in the namespace